docs/MODULES.md

```diff
@@ -68,18 +68,89 @@ For more information please see the [CJDNS FAQ](https://github.com/cjdelisle/cjd
+
 ## Yggdrasil
+
 Yggdrasil is another piece of mesh routing software similar to CJDNS, but with potentially better performance and more active development. For more info visit the [website](https://yggdrasil-network.github.io).
 
-### Yggdrasil subnetting
+## Yggdrasil subnetting
 
 Yggdrasil will give each node (like your Pi, for example) an IPv6 address, but it can also give each node a subnet to distribute to its clients. This means that if you connect to the WiFi of your Pi, your device can get a unique Yggdrasil address, with all the benefits it provides. These include being able to access your device directly, without being NATed or blocked.
 
 However, the Pi does have a firewall, so various commands need be run to allow access to clients. By default all Yggdrasil client access is blocked. See [**Firewall/IPv6/Yggdrasil Clients**](#yggdrasil-clients) to learn how to change that.
 
+### Yggdrasil Interent Peering
+
+Other peers can be found in the [public-peers](https://github.com/yggdrasil-network/public-peers) repo. Try to connect to only a few peers, and ones that are close to where you live.
+
+Some will be IPv4, others IPv6. Head over to your mesh node yet again, and enter the following in your terminal:
+
+```
+$ sudo nano /etc/yggdrasil.conf
+```
+
+We are interested in this section of the config file:
+
+```
+List of connection strings for static peers in URI format, e.g. tcp://a.b.c.d:e or socks://a.b.c.d:e/f.g.h.i:j.
+Peers: []
+```
+
+This is where we are going to insert the code to connect to the peer node. Your code should look similar to this:
+
+```
+Peers: ["tcp://11.22.33.44:1234"]
+```
+
+Exit out of nano and save the changes. Restart Yggdrasil with a `sudo killall yggdrasil` command. Pass a `status` command to terminal and you should see green text where Yggdrasil is printed with the words `[ACTIVE]` present. You are now connected to the remote peer with Yggdrasil. If you see`[INACTIVE]`, then you need to check your code
+for typos, make sure there are "" around the whole entire string.
+
 ## Yggdrasil IPTunnel
 
-This module will allow you to tunnel Internet from an EXIT node (server) that has Internet to another node that does not. To do this you must exchange public keys.  The public key can be found in /etc/yggdrasil.conf
+This module uses the [CKR](https://yggdrasil-network.github.io/2018/11/06/crypto-key-routing.html) function of yggdrasill to allow you to tunnel internet from an exit node (server) that has access to the internet to another node that does not. To do this you must exchange public keys.  The public key can be found in /etc/yggdrasil.conf
 
 To use this module you must have it installed. You can check to see if the file `/usr/local/sbin/yggdrasil-setup` exists. If it does then you have it installed, otherwise you need to have the module installed first.
```

Comment:
Explain how to install it maybe?

Reply:
You install by installing the module when installing the node.

At present you officially "CANT" install after-the fact. So i would rather not write that down.

changed to read

> To use this module you must have it installed. You can check to see if the file `/usr/local/sbin/yggdrasil-setup` exists. If it does then you have it enable it during the protoype install.

Reply:
Yeah, all I meant was noting that there is a module to be installed for this.